lib/install/management/delete.go

```diff
@@ -238,11 +238,34 @@ func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *config.Vir
 	var err error
 	var children []*vm.VirtualMachine
 
-	// TODO: GET CHILDREN EITHER BY FOLDER OR RP BASED ON DEPLOYMENT
-	if children, err = d.parentResourcepool.GetChildrenVMs(d.op, d.session); err != nil {
+	parentFolder, err := vmm.ParentInventoryFolder(d.op, d.session)
+	if err != nil {
 		return err
 	}
 
+	if parentFolder.Reference() == d.session.VMFolder.Reference() {
+		// get the children from the resource pool since it is an old vch
+		d.op.Debugf("Found older VCH, looking in the resource pool for delete targets")
```

Comment:
again...I don't feel like we should appear to endorse ageism here...just state we are looking in the resource pool...